src/content/self-hosted/install/releases.mdx

```diff
@@ -5,7 +5,46 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+
+## 1.4.0
+<TODO: insert release date here once it's published>
+
+### Features
+- Update the Destroy All UI 
```

Comment:
The feature here is that the logic of `destroy` commands is always executed when a namespace or preview is deleted

Reply:
@rberrelleza could you help ^^?

Reply:
Maybe something like 

"Deleting a namespace or preview will inherit the behavior from the `destroy` command"